iron-list.html

```diff
@@ -992,7 +992,8 @@
      */
     _resizeHandler: function() {
       this.debounce('resize', function() {
-        if (this._physicalItems && this._isVisible) {
+        this._render();
+        if (this._physicalItems && this._isVisible && this._initRendered) {

```

Comment:
Make `_initRendered` the first check and suggest changing this to `_itemsRendered`